partner_account_payment_mode_multicompany_default/models/partner.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo import models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_multicompany_account_payment_mode_key(self):
+        return "name"
+
+    def propagate_multicompany_customer_payment_mode(self):
+        self._propagate_multicompany_field("customer_payment_mode_id")
+
+    def propagate_multicompany_supplier_payment_mode(self):
+        self._propagate_multicompany_field("supplier_payment_mode_id")
+
+    def _propagate_property_fields(self):
+        """Propagate accounts to other companies always."""
```

Comment:
perhaps add on the comment that is Overwrite to propagate on the parent (if it's not i suppose it can be remove)

Reply:
Wrong string. Actually this is about payment_mode and not accounts